tests/unit/components/app-popover/AppPopover.spec.jsx

```diff
@@ -0,0 +1,52 @@
+import AppPopover from '~/components/app-popover/AppPopover'
+import { expect } from 'vitest'
+import { fireEvent, screen, render, waitFor } from '@testing-library/react'
```

Comment:
Import from libraries first 
```suggestion
import { expect } from 'vitest'
import { fireEvent, screen, render, waitFor } from '@testing-library/react'
import AppPopover from '~/components/app-popover/AppPopover'
```